### PR TITLE
storagenode/piecestore: move queue size defaults

### PR DESCRIPTION
**Pull request by littleskunk**
What: Move the queue size out of the code and use default values instead

Why: It is confusing to see the wrong default values in the help output. This is how it should look like:
```
      --storage2.delete-queue-size int                    size of the piece delete queue (default 10000)
      --storage2.delete-workers int                       how many piece delete workers (default 1)
```

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [x] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: storagenode/pieces/deleter.go [hunk 1]
```diff
@@ -43,14 +43,6 @@ type Deleter struct {
 
 // NewDeleter creates a new Deleter.
 func NewDeleter(log *zap.Logger, store *Store, numWorkers int, queueSize int) *Deleter {
-	if numWorkers == 0 {
```
Comment (stefanbenten) [inline storagenode/pieces/deleter.go:46]: You should add a check against <=0 here.

File: storagenode/pieces/deleter.go [hunk 2]
```diff
@@ -43,14 +43,6 @@ type Deleter struct {
 
 // NewDeleter creates a new Deleter.
 func NewDeleter(log *zap.Logger, store *Store, numWorkers int, queueSize int) *Deleter {
```
Comment (egonelbre) [inline storagenode/pieces/deleter.go:45]: I think you also need to update constants in testplanet.

File: storagenode/pieces/deleter.go (hunk 1 above)
Comment (littleskunk) [reply storagenode/pieces/deleter.go:46]: Why don't we validate all config options? Why should a user add a value <= 0? I don't think this is needed at all.

Comment (egonelbre) [reply storagenode/pieces/deleter.go:46]: I think it helps to detect mis-configuration (e.g. forgetting to update testplanet config)

**Pull request merged**